### DISCUSSION TITLE
PGP encryption of already encrypted mails?

### DISCUSSION THREAD

**Discussion by genseirin** [simple-login/app#2350] [Q&A] [answered]
What happens if someone sends an already PGP-encrypted mail to an alias, which has encryption enabled? Will SimpleLogin just forward the original encrypted block, as it came from the sender, or encrypt it again with the public key that I saved?

I couldn't find the answer in your documentation, sorry if I missed it.

**Comment by nguyenkims** [comment, marked as answer]
SL will encrypt again as it doesn't check whether the email is already encrypted.
